### DISCUSSION TITLE
Scheduled deploy. Is it possible?

### DISCUSSION THREAD

**Discussion by palnic-chisel** [vercel/community#2250] [Help] [answered]
### Summary

Hi, is it possible to schedule a deploy for an application every night for example?
Thank you,
Nicola

**Comment by amyegan** [comment, marked as answer]
There are a couple of ways I can think of to handle this (though I'm sure others can think of more)! The first is to use Vercel CLI to create a custom workflow with a CI tool of your choice. If using GitHub Actions, for example, you could use the [schedule](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) event to trigger your deployment at a set time.

The second is to use ISR. You could use background revalidation on a schedule or use on-demand revalidation with Cron Jobs to refresh content nightly. With this option you would not need to do a full deployment for content modifications but could update just the content that you expect to change.

Here are a few documentation pages that can help with whichever option you choose:
- [How can I use the Vercel CLI for custom workflows?](https://vercel.com/guides/using-vercel-cli-for-custom-workflows)
  - [How can I use GitHub Actions with Vercel?](https://vercel.com/guides/how-can-i-use-github-actions-with-vercel)
  - [How can I use GitLab Pipelines with Vercel?](https://vercel.com/guides/how-can-i-use-gitlab-pipelines-with-vercel)
  - [How can I use Bitbucket Pipelines with Vercel?](https://vercel.com/guides/how-can-i-use-bitbucket-pipelines-with-vercel)
- [Incremental Static Regeneration (ISR)](https://vercel.com/docs/concepts/incremental-static-regeneration/overview#)
  - [Background Revalidation](https://vercel.com/docs/concepts/incremental-static-regeneration/quickstart#background-revalidation)
  - [On-Demand Revalidation](https://vercel.com/docs/concepts/incremental-static-regeneration/quickstart#on-demand-revalidation)
- [How to Setup Cron Jobs on Vercel](https://vercel.com/guides/how-to-setup-cron-jobs-on-vercel)
